File: Conectify.Database/Models/Dashboard/DashboardDevice.cs
using Conectify.Database.Interfaces;

namespace Conectify.Database.Models.Dashboard;
public class DashboardDevice : IEntity
{
    public Guid Id { get; set; }
    public Guid DashBoardId { get; set; }

    public Guid DeviceId { get; set; }

    public int PosX { get; set; }

    public int PosY { get; set; }

    public string SourceType { get; set; } = null!;
}

File: Conectify.UI/src/models/Dashboard/DashboardDevice.ts
export interface DashboardDeviceApi {
    id: string;
    deviceId: string;
    posX: number;
    posY: number;
    sourceType: string;
}

export interface AddDashboardDeviceApi {
    deviceId: string;
    posX: number;
    posY: number;
    sourceType: string;
}